tests/foreman/ui/test_host.py

```diff
@@ -1088,9 +1085,7 @@ def test_positive_search_by_org(session, smart_proxy_location, target_sat):
 
 
 @pytest.mark.tier2
-def test_positive_validate_inherited_cv_lce(
-    session, module_host_template, target_sat, default_smart_proxy
-):
+def test_positive_validate_inherited_cv_lce(session, module_org, target_sat, module_host_template):
```

Comment:
You can remove the `module_org` from here I think as you have taken care of it with `module_host_template`